include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -968,7 +1101,7 @@ namespace VectorTools
       mass_matrix.vmult_add(rhs, inhomogeneities);
 
       //now invert the matrix
-      ReductionControl     control(rhs.size(), 0., 1e-12, false, false);
+      ReductionControl     control(5.*rhs.size(), 0., 1e-12, false, false);
```

Comment:
Roundoff errors in the conjugate gradient method in case there are bad meshes and/or bad shape functions. We have a bunch of such test cases. An example I recall uses `FE_Q_Hierarchical` at a degree four or five where conditioning gets so bad that we get into trouble. Now of course this indicates a delicate linear system to start with and just because the CG error estimate at some point beyond `rhs.size()` iterations is happy does not mean that the target tolerance is met - but I do not think that it should be addressed in this PR.